test/verify/check-login

```diff
@@ -75,6 +75,14 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         else:
             b.wait_text("#login-error-message", "Permission denied")
 
+        # Try to login with disabled shell; this does not work on Atomic where
+        # we log in through ssh
+        if "atomic" not in os.environ.get("TEST_OS", ""):
```

Comment:
Lets just use:

```
if m.image not in [ "continuous-atomic", "fedora-atomic", "rhel-atomic" ]:
```

We've been bitten many times by overgeneralized in matching the operating system. As we add new operating systems, we want to explicitly add them to lists like this.